tests/manage/pv_services/pvc_clone/test_pvc_to_pvc_clone.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version, ManageTest, tier1
+)
+from ocs_ci.ocs.resources import pod
+from tests import helpers
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@skipif_ocs_version('<4.6')
+@pytest.mark.parametrize(
+    argnames=["interface_type"],
+    argvalues=[
+        pytest.param(
+            constants.CEPHBLOCKPOOL, marks=pytest.mark.polarion_id("")
+        )
+    ]
+)
+class TestPvcToPvcClone(ManageTest):
+    """
+    Tests to verify PVC to PVC clone feature
+    """
+    @pytest.fixture(autouse=True)
+    def setup(self, interface_type, storageclass_factory, pvc_factory, pod_factory):
+        """
+        create resources for the test
+        Args:
+            interface_type(str): The type of the interface
+                (e.g. CephBlockPool, CephFileSystem)
+            storageclass_factory: A fixture to creare new storage class
+            pvc_factory: A fixture to create new pvc
+            pod_factory: A fixture to create new pod
+        """
+        self.sc_obj = storageclass_factory(interface=interface_type)
+        self.pvc_obj = pvc_factory(
+            interface=interface_type,
+            size=1,
+            status=constants.STATUS_BOUND
+        )
+        self.pod_obj = pod_factory(
+            interface=interface_type,
+            pvc=self.pvc_obj,
+            status=constants.STATUS_RUNNING
+        )
```

Comment:
To make it synchronous with `pvc-clone.yaml` on ceph-csi repo [here](https://github.com/ceph/ceph-csi/blob/master/examples/rbd/pvc-clone.yaml) ... It may be better to leave it at 1Gi. Any thoughts?